crates/bevy_reflect/src/std_traits.rs

```diff
@@ -0,0 +1,20 @@
+use crate::{FromType, Reflect};
+
+#[derive(Clone)]
+pub struct ReflectDefault {
```

Comment:
Nit: I know it's a relatively straightforward struct, but I think a doc comment could still be nice.